profiles/common/modules/features/solr_config/solr_config.module

```diff
@@ -6,26 +6,34 @@
 
 include_once 'solr_config.features.inc';
 
+/**
+ * Implements hook_apachesolr_exclude().
+ */
 function solr_config_apachesolr_exclude($entity_id, $entity_type, $row, $env_id) {
   if ($entity_type == 'file') {
     $node = node_load($row->parent_entity_id);
 
-    if (isset($node->workbench_moderation)) { // node moderate with workbench moderation
+    if (isset($node->workbench_moderation)) {
+      // Node moderate with workbench moderation.
       $workflow = $node->workbench_moderation;
-      if (isset($workflow['published'])) { // node parent is published
+      if (isset($workflow['published'])) {
+        // Node parent is published.
         return FALSE;
       }
       else {
-        apachesolr_index_delete_entity_from_index($env_id, $entity_type, $entity_id . "-node-" . $node->nid); // remove file from solr index
+        apachesolr_index_delete_entity_from_index($env_id, $entity_type, $entity_id . "-node-" . $node->nid);
+        // Remove file from solr index.
```

Comment:
This is a bug in the PHPCBF I think, where it puts inline documentation on the next line instead of on the line above. The documentation should be one line higher.